### PR TITLE
[15.0][MIG] contract_queue_job

### PR DESCRIPTION
**Pull request by palkesh-aktivsoftware**
Standard Migration

### CODE REVIEW

**Review by pablontura — APPROVED** [review]
LGTM

**Review by juliap98 — APPROVED** [review]
LGTM

**Comment by rousseldenis** [comment]
@palkesh-aktivsoftware Thanks for this. Could you rename your last commit with something like '[15.0][MIG] contract_queue_job' ?

**Comment by palkesh-aktivsoftware** [comment]
> @palkesh-aktivsoftware Thanks for this. Could you rename your last commit with something like '[15.0][MIG] contract_queue_job' ?

sure, I made this change. Please review it.

File: contract_queue_job/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "summary": """
         This addon make contract invoicing cron plan each contract in a job
         instead of creating all invoices in one transaction""",
-    "version": "12.0.1.1.0",
+    "version": "15.0.1.1.0",
```
Comment (rousseldenis) [inline contract_queue_job/__manifest__.py:9]: You should reset minor version too

15.0.1.0.0

**Comment by ygol** [comment]
How may I access a runboat instance for this PR? 
I would like to check if I can reproduce that invoices are created every time the cron is launched resulting in multiple time the same draft invoice with the invoice date = when the cron was launched:
image

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**